kitchen.dokken.yml

```diff
@@ -1,35 +1,37 @@
 ---
 driver:
   name: dokken
-  chef_version: 14.12.9
+  chef_version: :latest
   privileged: true # because Docker and SystemD/Upstart
 
 transport:
   name: dokken
 
 lifecycle:
+  # The purpose of this code is to build the InSpec gems from source and then make them available
+  # to the install_inspec cookbook, which will install the gems. This means that the inspec used
```

Comment:
I'm confused how this works - these commands are setting up for the `install_inspec` recipe to install the current InSpec on the test instance. But kitchen's inspec verifier runs the local copy of InSpec against the test instance as a target.

Am I missing some kitchen magic here where the InSpec installed on the instance is used instead of the local one?

I can't quickly test this locally because I'm getting this error (which I suspect is something dumb/untidy in my local environment)

```
-----> Converging <resources-ubuntu-2004>...
       Creating kitchen sandbox in /Users/jstocks/.dokken/kitchen_sandbox/fe0834e046-resources-ubuntu-2004
       Preparing dna.json
       Resolving cookbook dependencies with Berkshelf 7.0.10...
       Removing non-cookbook files before transfer
       Preparing validation.pem
       Preparing client.rb
/opt/chef/embedded/lib/ruby/2.7.0/rubygems/dependency.rb:313:in `to_specs': Could not find 'inspec-core' (= 4.23.4) - did find: [inspec-core-4.23.10] (Gem::MissingSpecVersionError)
Checked in 'GEM_PATH=/root/.gem/ruby/2.7.0:/opt/chef/embedded/lib/ruby/gems/2.7.0', execute `gem env` for more information
	from /opt/chef/embedded/lib/ruby/2.7.0/rubygems/dependency.rb:323:in `to_spec'
	from /opt/chef/embedded/lib/ruby/2.7.0/rubygems/core_ext/kernel_gem.rb:62:in `gem'
	from /opt/chef/bin/chef-client:63:in `<main>'

```